crypt4gh/aws.py

```diff
@@ -0,0 +1,38 @@
+
+import boto3
+import logging
+import sys
+
+"""
+Handy dandy utilities for dealing with AWS resources
+"""
+
+log = logging.getLogger(__name__)
```

Comment:
could you rename `log` into `LOG`, just for consistency with the other modules?

Reply:
Done.